### PR TITLE
Readme.md improvements 

### LINKED ISSUE #366: Add more info
## Summary

Can we add the discord channels to the info in [join mainnet](https://github.com/cosmos/gaia/blob/master/docs/gaia-tutorials/join-mainnet.md#add-seed-nodes) docs?

## Problem Definition
Recently we are using Discord rather than Riot.
The people can find all the chats/channels to organise in one document.

## Proposal

Add some Discord channels:

- [Introduce yourself](https://discordapp.com/channels/669268347736686612/669268347736686615)

- [General](https://discordapp.com/channels/669268347736686612/669275164999155742)

- [Announcements](https://discordapp.com/channels/669268347736686612/669488112807706625)

- [Validators](https://discordapp.com/channels/669268347736686612/669274895041036349)
____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

### LINKED ISSUE #378: Easier Mainnet Nodes
*Goal*:  Reduce setting up a mainnet node to the fewest number of steps possible without breaking any existing practices

*Context*:

https://twitter.com/gadikian/status/1254828427728674817?s=20

This week I put up a server at hetzner.de to run a decentralized VPN node, because hetzner has cheap unlimited bandwidth and the connection is really excellent.  I was left with a ton of excess storage, RAM, and CPU capacity.  

I decided that I'd run nodes for all of the chains that I was interested in running nodes for, and got an opportunity to compare the setup process between blockchain ecosystems.  

For Cosmos, what I found was that the default path of "I just want to run a mainnet node" was a bit tougher than it should be and the docs/links are possibly a little dated.  Some of the seeds referenced on the launch repo are no longer up, for example. 

So once I'm done typing up this issue, I'm going to make a PR that attempts to solve this by: 

* provide a file called `seeds`
* provide the user with some bash in the "quick start" process that ensures that they do not need to move the seeds into config.toml manually
* Putting exact directions in the quick start, which result in a functioning node when followed exactly and do not require much interpretation, editing of config files, or thought to get a fully functioning mainnet node

### PR DESCRIPTION
**Pull request by faddat**
Closes: #378 
Closes: #366 

## Description

This:

* Defines Gaia
* Defines Cosmos
* Defines IBC

And how they relate to each other.

  It also adds a file called `seeds` which I pulled from my mainnet node `fortress`.  I would actually like to update that file to contain the 100 longest running mainnet nodes, but don't know where to source that data.  

It also provides a clear build path for the most common task people will want to do when viewing readme.md -- setting up a mainnet node on Mac/Linux.  

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Comment by faddat** [comment]
Squashing now...

I think :).

File: seeds [hunk 1]
```diff
@@ -0,0 +1 @@
+ba3bacc714817218562f743178228f23678b2873@5.83.160.108:26656,1e63e84945837b026f596ed8ae68708783d04ad4@51.75.145.123:26656,d2d452e7c9c43fa5ef017552688de60a5c0053ee@34.245.217.163:26656,dd36969b56c740bb40bb8badd4d4c6facc35dc24@206.189.115.41,a0aca8fb801c69653a290bd44872e8457f8b0982@47.99.180.54,27f8dd3bdbecbef7192291083706c156e523d8e0@3.122.248.21,aee0df1a660f301d456a0c2f805b372f7341e8ec@63.35.230.143:26656,7d1f660b361d6286715c098a3a171e554e9642bb@34.254.205.37,fa105c2291ac4aa452552fa4835266300a8209e1@88.198.41.62,bd410d4564f7e0dd9a0eb16a64c337a059e11b80@47.103.35.130
```
Comment (alexanderbez) [inline seeds]: Why do we have a seeds file?

File: README.md [hunk 2]
```diff
@@ -8,26 +8,32 @@
...
-
-## Quick Start
+This assumes that you're running Linux or MacOS and have installed [Go 1.14+](https://golang.org/dl/).  It will build and install Gaia, allow you to name your node, add seeds to your config file, start your node and use gaiacli to check the status of your node.  Welcome to the Cosmos!
 
 ```
+git clone -b v2.0.9 https://github.com/cosmos/gaia
+cd gaia
 make install
+gaiad init yournodenamehere
+SEEDS=$(cat seeds); original_string="seeds = \"\""; replace_string="seeds = \"$SEEDS\""; sed -i -e "s/$original_string/$replace_string/g" "$HOME/.gaiad/config/config.toml"
```
Comment (alexanderbez) [inline README.md]: Just list the seeds in the README file. I think that'll read better and it'll be easier to maintain 👍

File: seeds (hunk 1 above)
Comment (faddat) [reply seeds]: The seeds file was to avoid having a big old ugly list of seeds in readme.md, more or less.

File: README.md (hunk 2 above)
Comment (faddat) [reply README.md]: You're right.  This is simpler, though it does add another step.  

Another reason for the seeds file is that I wanted to put a hundred seeds in there.  

At the same time, I'm sure we can all agree that's some pretty ugly bash and we may be better off without it.  

How about no seeds file, 5 seeds listed right in readme.md?

Comment (faddat) [reply README.md]: Goal with putting seeds here (in a seeds file or in the readme) is to allow a fresh user to out up a node without navigating away from readme.md

Comment (alexanderbez) [reply README.md]: Yes, I understand the goal. Just limit the seeds to 10-15 or so 👍

**Comment by tac0turtle** [comment]
This looks good. I agree for now we should have the seeds in the readme. At some point we should add a section in the docs with seeds and nodes. We will need a page with rpc's when state sync reaches the hub as well

File: README.md (hunk 2 above)
Comment (faddat) [reply README.md]: OK, I'll make the changes now, thanks.

**Comment by faddat** [comment]
Most recent change was to describe each step a little more effectively.  Here's another option we could use to let users know exactly what each step does by putting comments in the code-fenced bash:

```bash
# Fetch Genesis
curl https://raw.githubusercontent.com/cosmos/launch/master/genesis.json > $HOME/.gaiad/config/genesis.json
# Start Your Node
gaiad start
# Check your node with gaiacli
gaiacli status
```

**Comment by tac0turtle** [comment]
yes we should do this, makes it nicer

**Comment by faddat** [comment]
I'll do that, then :).

**Review by tac0turtle — APPROVED** [review]
LGTM

love the additions!! Thank you for this!!

**Review by alessio — APPROVED** [review]
ACK from me

File: README.md [hunk 3]
```diff
@@ -8,26 +8,74 @@
...
+aee0df1a660f301d456a0c2f805b372f7341e8ec@63.35.230.143:26656
+7d1f660b361d6286715c098a3a171e554e9642bb@34.254.205.37
+fa105c2291ac4aa452552fa4835266300a8209e1@88.198.41.62
+bd410d4564f7e0dd9a0eb16a64c337a059e11b80@47.103.35.130
+```
+
+Download Genesis, Start your Node, Check your Node Status
+```bash
+# Download genesis.json
+curl https://raw.githubusercontent.com/cosmos/launch/master/genesis.json > $HOME/.gaiad/config/genesis.json
```
Comment (alessio) [inline README.md]: `wget https://raw.githubusercontent.com/cosmos/launch/master/genesis.json` should be enough (and in some bare metail images `curl` may not even be available). But it's not a big deal at all.

Comment (faddat) [reply README.md]: you're right :).  

I'll make this a wget situation.

**Pull request merged**